# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 70eab0533b00
# Full Hash: 70eab0533b00cee4ac31f7e3250d78d2a9a09758
# Author: Ray Kraesig <rkraesig@mozilla.com>
# Date: 2023-09-16 09:14:45
# Regressor Bug: 1837079
# File Overlap Count: 1
# Description:
#   Bug 1837079 - [7/10] Open Windows file picker out-of-process  r=gstoll,handyman,ipc-reviewers,nika
#   
#   When opening a new Windows file dialog, open it out-of-process if
#   possible. Fall back to opening it in-process if that fails. (This
#   behavior is configurable with a pref.)
# ==============================================================================

diff -r 488707a84723 -r 70eab0533b00 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Fri Sep 15 18:10:01 2023 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Fri Sep 15 18:10:01 2023 +0000
@@ -15553,6 +15553,19 @@
   value: true
   mirror: always
 
+# Whether to open the Windows file and folder pickers "remotely" (in a utility
+# process) or "locally" (in the main process).
+#
+# Valid values:
+#  *  0: auto (possibly release-channel-dependent)
+#  *  1: remotely, falling back to locally
+#  *  2: remotely, no fallback
+#  * -1: locally, no fallback
+- name: widget.windows.utility_process_file_picker
+  type: RelaxedAtomicInt32
+  value: -1
+  mirror: always
+
 # The number of messages of each type to keep for display in
 # about:windows-messages
 - name: widget.windows.messages_to_log